label_studio/core/feature_flags/stale_feature_flags.py

```diff
@@ -29,4 +29,22 @@
     'fflag_feat_front_prod_292_archive_workspaces_short': True,
     'fflag_feat_all_lsdv_4915_async_task_import_13042023_short': True,
     'fflag_fix_all_lsdv_4971_async_reimport_09052023_short': True,
+    # Jan 16
+    'fflag_fix_front_leap_443_select_annotation_once': False,
```

Comment:
I need to investigate why is it false, that was a very good improvement for both UX and performance. Remove it for now from this list, please
```suggestion
```